scrapy/commands/parse.py

```diff
@@ -206,7 +209,9 @@ def callback(response, **cb_kwargs):
                     req.meta['_depth'] = depth + 1
                     req.meta['_callback'] = req.callback
                     req.callback = callback
-                return requests
+                return items + requests
+            else:
+                return items  # dumps the remaining output to a FILE
```

Comment:
Maybe we should only do this when the new option is used, to minimize behavior changes?

Reply:
Yes, that will be better. I will do that now